azurerm/internal/services/compute/virtual_machine_scale_set.go

```diff
@@ -807,6 +819,14 @@ func ExpandVirtualMachineScaleSetDataDisk(input []interface{}) *[]compute.Virtua
 			}
 		}
 
+		if iops := raw["disk_iops_read_write"].(int); iops != 0 {
```

Comment:
should we be validating this isn't set when the disk type is set to something other then an ultra SSD?

Reply:
Will do